crates/chia-consensus/src/gen/validation_error.rs

```diff
@@ -95,39 +204,145 @@ pub fn first(a: &Allocator, n: NodePtr) -> Result<NodePtr, ValidationErr> {
+            ErrorCode::NotBlockButHasData => 49,
+            ErrorCode::IsTransactionBlockButNoData => 50,
+            ErrorCode::InvalidPrevBlockHash => 51,
+            ErrorCode::InvalidTransactionsInfoHash => 52,
+            ErrorCode::InvalidFoliageBlockHash => 53,
+            ErrorCode::InvalidRewardCoins => 54,
+            ErrorCode::InvalidBlockCost => 55,
+            ErrorCode::NoEndOfSlotInfo => 56,
+            ErrorCode::InvalidPrevChallengeSlotHash => 57,
+            ErrorCode::InvalidSubEpochSummaryHash => 58,
+            ErrorCode::NoSubEpochSummaryHash => 59,
+            ErrorCode::ShouldNotMakeChallengeBlock => 60,
+            ErrorCode::ShouldMakeChallengeBlock => 61,
+            ErrorCode::InvalidChallengeChainData => 62,
+            ErrorCode::InvalidCcEosVdf => 65,
+            ErrorCode::InvalidRcEosVdf => 66,
+            ErrorCode::InvalidChallengeSlotHashRc => 67,
+            ErrorCode::InvalidPriorPointRc => 68,
+            ErrorCode::InvalidDeficit => 69,
+            ErrorCode::InvalidSubEpochSummary => 70,
+            ErrorCode::InvalidPrevSubEpochSummaryHash => 71,
+            ErrorCode::InvalidRewardChainHash => 72,
+            ErrorCode::InvalidSubEpochOverflow => 73,
+            ErrorCode::InvalidNewDifficulty => 74,
+            ErrorCode::InvalidNewSubSlotIters => 75,
+            ErrorCode::InvalidCcSpVdf => 76,
+            ErrorCode::InvalidRcSpVdf => 77,
+            ErrorCode::InvalidCcSignature => 78,
+            ErrorCode::InvalidRcSignature => 79,
+            ErrorCode::CannotMakeCcBlock => 80,
+            ErrorCode::InvalidRcSpPrevIp => 81,
+            ErrorCode::InvalidRcIpPrevIp => 82,
+            ErrorCode::InvalidIsTransactionBlock => 83,
+            ErrorCode::InvalidUrsbHash => 84,
+            ErrorCode::OldPoolTarget => 85,
+            ErrorCode::InvalidPoolSignature => 86,
+            ErrorCode::InvalidFoliageBlockPresence => 87,
+            ErrorCode::InvalidCcIpVdf => 88,
+            ErrorCode::InvalidRcIpVdf => 89,
+            ErrorCode::IpShouldBeNone => 90,
+            ErrorCode::InvalidRewardBlockHash => 91,
+            ErrorCode::InvalidMadeNonOverflowInfusions => 92,
+            ErrorCode::NoOverflowsInFirstSubSlotNewEpoch => 93,
+            ErrorCode::MempoolNotInitialized => 94,
+            ErrorCode::ShouldNotHaveIcc => 95,
+            ErrorCode::ShouldHaveIcc => 96,
+            ErrorCode::InvalidIccVdf => 97,
+            ErrorCode::InvalidIccHashCc => 98,
+            ErrorCode::InvalidIccHashRc => 99,
+            ErrorCode::InvalidIccEosVdf => 100,
+            ErrorCode::InvalidSpIndex => 101,
+            ErrorCode::TooManyBlocks => 102,
+            ErrorCode::InvalidCcChallenge => 103,
+            ErrorCode::InvalidPrefarm => 104,
+            ErrorCode::AssertSecondsRelativeFailed => 105,
+            ErrorCode::BadCoinbaseSignature => 106,
+            ErrorCode::NoTransactionsWhileSyncing => 108,
```

Comment:
I suppose you compensate by skipping 107 here. I think you should add a comment here too though. It looks like a mistake and a future reader of the code may "fix" it.

Reply:
Sure